DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -3184,6 +3210,13 @@ defaultproperties
     bingo_options(320)=(event="M05MeetJaime_Played",desc="Talk to Jaime during the escape",max=1,missions=32)
     bingo_options(321)=(event="jughead_Dead",desc="Kill El Rey",max=1,missions=8)
     bingo_options(322)=(event="JoshuaInterrupted_Played",desc="He was the one who wanted to be a soldier",max=1,missions=1024)
+    bingo_options(323)=(event="LebedevLived",desc="Keep Lebedev alive",max=1,missions=8)
+    bingo_options(324)=(event="AimeeLeMerchantLived",desc="Let Aimee and Le Merchant live",max=1,missions=1536)
+    bingo_options(325)=(event="OverhearLebedev_Played",desc="This socket is being monitored",max=1,missions=8)
+    bingo_options(326)=(event="ThugGang_AllianceDead",desc="Slaughter the Rooks",max=10,missions=8) // there are ordinarily 11 Rooks
+    bingo_options(327)=(event="GiveZyme",desc="Who needs Rock?",max=2,missions=8) // Huh?  Not me.  He could just die.  Take his fifty-cut zyme and blow.
+    // bingo_options(328)=(event="MarketKid_BindNameUnconscious",desc="Crime doesn't pay",max=1,missions=64)
```

Comment:
we like to know the number of bingo events we have, so don't put a number for commented out goals

```suggestion
    // bingo_options()=(event="MarketKid_BindNameUnconscious",desc="Crime doesn't pay",max=1,missions=64)
```